.github/ISSUE_TEMPLATE/ql--false-positive.md

```diff
@@ -0,0 +1,36 @@
+---
+name: False positive
```

Comment:
```suggestion
name: CodeQL False positive
```

Reply:
I wondered about having a `CodeQL` prefix in the title, but decided against it because the issue template is for the `github/codeql` repository, so `codeql` is sort of implied. Happy to add the prefix though, if you think it is important here.

Reply:
I wanted to re-emphasize this here because it's the bold text users are likely to read. Sometimes general code scanning users end up here and it helps to highlight that this is about CodeQL again I think